### PR TITLE
362 add spotfiy button to skeletons and make it darker

### LINKED ISSUE #362: add spotfiy button to skeletons and make it darker

### PR DESCRIPTION
**Pull request by Iqro-dev**
closes #362

### CODE REVIEW

File: app/components/common/spotify-link.tsx [hunk 1]
```diff
@@ -11,9 +15,15 @@ import {
 export interface SpotifyLinkProps extends Omit<LinkProps, 'href'> {
   href?: string
   className?: string
+  skeleton?: boolean
```
Comment (Mnigos) [inline app/components/common/spotify-link.tsx:18]: rename it to `isDisabled`

File: app/components/common/spotify-link.tsx [hunk 2]
```diff
@@ -25,13 +35,14 @@ export function SpotifyLink({ href, className, ...props }: SpotifyLinkProps) {
...
               {...props}
             >
-              <Image
-                src="/spotify-icon-white.png"
-                width={20}
-                height={20}
-                alt={''}
-                className="min-w-[20px] h-[20px]"
-              />
+              <IconContext.Provider
```
Comment (Mnigos) [inline app/components/common/spotify-link.tsx:38]: Can't you just set color of this icon by setting `text-gray` in `className`?

**Review by Mnigos — CHANGES_REQUESTED** [review]

File: app/components/common/spotify-link.tsx (hunk 2 above)
Comment (Iqro-dev) [reply app/components/common/spotify-link.tsx:38]: that's how it was in the documentation, i dont know

Comment (Iqro-dev) [reply app/components/common/spotify-link.tsx:38]: i'd rather leave it like that, removing it could cause bugs in the future

**Review by Mnigos — APPROVED** [review]

**Pull request merged**